src/app/shared/branding.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { Fabric8UIConfig } from '../shared/config/fabric8-ui-config';
+
+import openshiftLogo from '../../assets/images/OpenShift-io_logo.png';
+import fabric8Logo from '../../assets/images/fabric8_logo.png';
+
+@Injectable()
+export class BrandingService {
+  private isFabric8: boolean;
+  private _logo: string;
```

Comment:
Based on this configuration, these private variable are now READONLY. They can only be set by the constructor. If we ever need to set this, they will need Setter methods.